src/services/apimap-field-builder.js

```diff
@@ -139,7 +139,24 @@ function ApimapFieldBuilder(model, column, options) {
       // NOTICE: Do not use the primary keys default values to prevent issues with UUID fields
       //         (defaultValue: DataTypes.UUIDV4).
       } else if (!_.includes(_.keys(model.primaryKeys), column.fieldName)) {
-        schema.defaultValue = column.defaultValue;
+        // FIXME: `column.defaultValue instanceof Sequelize.Utils.Literal` fails for unknown reason.
+        // NOTICE: Remove Sequelize.Utils.Literal wrapper to display actual value in UI.
+        if (_.isObject(column.defaultValue) && (column.defaultValue.constructor.name === 'Literal')) {
+          // NOTICE: do not export literal default values to UI by default
+          schema.defaultValue = undefined;
+
+          // NOTICE: Keep only simple values, and hide expressions.
+          const defaultValue = column.defaultValue.val;
+          if ((_.isBoolean(defaultValue))
+           || (_.isNumber(defaultValue))
+           || (['true', 'false'].includes(defaultValue.toLowerCase()))
```

Comment:
Don't think so. As `val` is a string in `Literal`, `true` is a "true" boolean, while `\'true\'` is a string containing `true`. This might be interpreted differently by the SQL server.